tensorrt_llm/_torch/custom_ops/cute_dsl_custom_ops.py

```
@@ -2809,6 +2854,48 @@ def cute_dsl_nvfp4_grouped_gemm_finalize_inplace_blackwell(
         )
         runner(inputs, tactic=best_tactic)
 
+    @torch.library.register_fake(
+        "trtllm::cute_dsl_nvfp4_grouped_gemm_finalize_inplace_blackwell")
+    def _(
+        input: torch.Tensor,
```

Ruff (0.16.1)

[error] Function argument `input` is shadowing a Python builtin

(A002)